monai/transforms/transforms.py

```diff
@@ -80,6 +81,65 @@ def __call__(self, img):
         return np.flip(img, self.axis)
 
 
+@export
+class Zoom:
+    """ Zooms a 3d image. Uses scipy.ndimage.zoom or cupyx.scipy.ndimage.zoom in case of gpu. 
```

Comment:
This transform can only be applied to 3D image?
Thanks.

Reply:
No. that's a mistake. I'll correct it.